jove.py

```diff
@@ -76,7 +76,7 @@ def join_if_possible(self, regions, forward):
 
         def get_sample(self):
             text = self.regions[0][0:256]
-            text = text.strip("\n \t").replace("\n", "\\n")
+            text = text.strip("\n \t").replace("\n", " \\n")
```

Comment:
I did this in my implementation in registers because think it is easier to visualize where new lines begin so helps with the flow of text but maybe you disagree.